tests/test_migrate_points.cairo

```diff
@@ -234,6 +225,66 @@ fn test_migrate_points_initiated_by_DAO() {
     
 }
 
+#[test]
+#[should_panic(expected: ('Caller is not the owner', ))]
+fn test_migrate_points_initiated_by_DAO_not_owner() { 
+    let (master_address, dev_guildSBT_address, design_guildSBT_address, marcom_guildSBT_address, problem_solving_guildSBT_address, research_guildSBT_address) = deploy_contracts_and_initialise();
+    let (user1_contribution, user2_contribution) = update_contribution_and_minting_sbt(master_address, dev_guildSBT_address, design_guildSBT_address, marcom_guildSBT_address, problem_solving_guildSBT_address, research_guildSBT_address);
+
+    let master_dispatcher = IMasterDispatcher { contract_address: master_address };
+    
+    // migrating user1 -> user3 and user2 -> user4
+    let mut old_addresses = ArrayTrait::new();
+    old_addresses.append(user1());
+    old_addresses.append(user2());
+
+    let mut new_addresses = ArrayTrait::new();
+    new_addresses.append(user3());
+    new_addresses.append(user4());
+
+    let mut spy = spy_events(SpyOn::One(master_address));
+
+    start_prank(master_address, user1());
+    master_dispatcher.migrate_points_initiated_by_DAO(old_addresses, new_addresses);
```

Comment:
Here's the exception raised and the code below is not executed. That's why I'm a fan of a safe dispatcher approach because you can wrap the exact line with the error. You've already used it in the test_mint file.